backend/__init__.py

```diff
@@ -108,12 +117,13 @@ class User(DB.Document, UserMixin):
     bio = DB.StringField(max_length=255)
     bookmarks = DB.ListField(UUIDField(), default=[])
     roles = DB.ListField(DB.ReferenceField(Role), default=[])
+    notifications = DB.ListField(EmbeddedDocumentField(Notification), default=[])
 
 
 class EmailConverter(BaseConverter):
-    regex = r"([a-z0-9!#$%&'*+\/=?^_`{|}~-]+(?:\." +\
-            r"[a-z0-9!#$%&'*+\/=?^_`"r"{|}~-]+)" +\
-            r"*(@|\sat\s)(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?" +\
+    regex = r"([a-z0-9!#$%&'*+\/=?^_`{|}~-]+(?:\." + \
+            r"[a-z0-9!#$%&'*+\/=?^_`"r"{|}~-]+)" + \
+            r"*(@|\sat\s)(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?" + \
```

Comment:
have no idea, pycharm did it automatically with reformat code i guess. But anyway pep8 didn't complain